controllers/authors.js

```diff
@@ -0,0 +1,31 @@
+const models = require('../models')
+
+const getAllAuthors = async (request, response) => {
+  const authors = await models.authors.findAll()
+
+  return response.send(authors)
+}
+
+const getAuthorByIdOrName = async (request, response) => {
+  const { identifier } = request.params
+
+  const author = await models.authors.findOne({
+    where: {
+      [models.Sequelize.Op.or]: [
+        { id: identifier },
+        { nameLast: { [models.Sequelize.Op.like]: `%${identifier}` } },
```

Comment:
This should have a `%` on both sides. With this on the front it means that the author's lastname has to _end_ with whatever was sent in. Specifically the example in the README doesn't work because we are searching for `wild` and expecting `Wilde` to show up.